supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc

```diff
@@ -83,18 +74,18 @@ For documentation questions, contact rhos-docs@redhat.com.
 [discrete]
 [[red-hat-openstack-platform]]
 ==== image:images/yes.png[yes] Red Hat OpenStack Platform (noun)
-*Description*: Spell out in full. This product name applies to Red Hat OpenStack Platform 8 and later versions.
+*Description*: On first use in a module, use the complete product name and the abbreviation in parentheses: _Red Hat OpenStack Platform (RHOSP)_. After the first instance, use _RHOSP_. This product name applies to RHOSP 8 and later versions. If you need to use the indefinite article before _RHOSP_, use _a_ not _an_.
```

Comment:
@ndeevy What is the correct format for referring to products/ emphasizing products? In this guide quotation marks are sometimes used to refer or emphasize a product name (see line 85 https://github.com/redhat-documentation/supplementary-style-guide/commit/edb628bb4a145d3fbb600e794f98590b7d030f12)

Should we use quotations or italics? I think the convention is to use quotation marks for product names and italics for published book/ guide titles. WDYT?

Reply:
Thanks for bringing this up, I wasn't sure myself and went with what was there previously without giving it enough thought. From a scan through, although there are some inconsistencies, most product glossaries do use double quotations, so I'm inclined to change a lot of these on that basis.

- AMQ uses italics, like OpenStack does/did: https://redhat-documentation.github.io/supplementary-style-guide/#destination
- Ceph uses double quotations: https://redhat-documentation.github.io/supplementary-style-guide/#placement-group
- Rhel, it seems that in deployment-scanner subsystem, it's double quotes for the terminology but single for the plural 's', which makes sense to me.
- OpenShift uses italics but only for the related terms mentioned in the description: https://redhat-documentation.github.io/supplementary-style-guide/#action
- Satellite uses single quotations: https://redhat-documentation.github.io/supplementary-style-guide/#capsule-server

In the documentation we use italics only for a definition -not for general emphasis, so I'd vote for that approach across this guide. For this PR though, I'll update it according to the majority of products with double quotations as you suggest @joaedwar.
thanks again :-]

Reply:
Okidoke, @joaedwar have another look please.
I change the entries with two exceptions. Here because it's a definition:

`Description: An _instance_ is a running virtual machine, ..`

and here for the articles:
`If you need to use the indefinite article before "RHOSP", use 'a' not 'an' ..`

Reply:
Thanks @ndeevy for the thorough explanation and for implementing the changes. You are wonderful and so is this PR :]